webpack.config.js

```diff
@@ -22,6 +22,20 @@ module.exports = {
       bcoin: path.resolve(__dirname, 'node_modules/bcoin/lib/bcoin-browser')
     }
   },
+
+  module: {
+    loaders: [
+      {
+        test: /\.jsx?$/,
+        exclude: /node_modules/,
+        loader: 'babel-loader',
+        query: {
+          presets:[ 'es2015', 'react', 'stage-2' ]
```

Comment:
Probably want to prefer [babel-preset-env](https://babeljs.io/docs/en/babel-preset-env) over `es2015`. What's in `stage-2` that we need here?

Reply:
`stage-2` has a lot of nice things in it that could also be useful for the future and I don't think it increases the bundle size

Reply:
Anything specific we're using though that's not available in `env`?

Reply:
the reason I ask is that `env` I believe includes everything up to es2017 and so could preclude the need for `stage-2` even if it doesn't increase bundle size, it is still another dependency and increase complexity.

Reply:
Updated 👍